문제1204번.py

```diff
@@ -0,0 +1,13 @@
+T=int(input())
+
+for s in range(T):
+    n= int(input())
+    lst=list(map(str,input().split()))
+    d={}
+    for j in lst:
+        if j in d:
+            d[j]+=1
+        else:
+            d[j]=1
+    d_arr=dict(sorted(d.items(), key=lambda x:x[1], reverse=True)) 
+    print(f'#{s+1} {list(d_arr)[0]}')
```

Comment:
`sorted` 함수는 리스트를 반환하는, 정렬해주는 함수입니다.
`dict` 함수는 입력 값을 딕셔너리 구조로 바꿔주는 함수입니다.
`list` 함수는 입력 값은 리스트 구조로 바꿔주는 함수입니다. 이때 입력 값이 딕셔너리라면, 딕셔너리의 key 값들로 이루어진 리스트를 반환합니다.

여기서 주의할 점이, 딕셔너리는 정렬되어 있지 않은 자료구조입니다. 이 문제에서는 괜찮았으나, 다른 테스트 케이스들이 더 주어졌다면 오답이 나올 수 있습니다. 즉 `d_arr`를 정렬 후 다시 딕셔너리로 바꾸는 작업을 하시면 정렬이 사라집니다!
```suggestion
    d_arr=sorted(d.items(), key=lambda x:x[1], reverse=True)
    print(f'#{s+1} {d_arr[0][0]}')
```
위는 올바르게 수정하는 방법 중 하나 입니다. 😄